Vagrantfile

```diff
@@ -6,6 +6,196 @@ $NODEMEM=256
 # Overwrite host locale in ssh session
 ENV["LC_ALL"] = "en_US.UTF-8"
 
+BOOTSTRAP_SCRIPT=<<__EOF__
+#!/bin/bash
+# Bootstrap the Ansible Lightbulb workshop environment on a Vagrant box
+
+#
+# Bash shell settings: exit on failing commands, unbound variables
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+
+#
+# Variables
+#
+# Color definitions
+readonly reset='\e[0m'
+readonly red='\e[0;31m'
+readonly yellow='\e[0;33m'
+readonly cyan='\e[0;36m'
+
+readonly workshop_user=vagrant
+
+readonly workshop_key="/home/${workshop_user}/.ssh/id_rsa"
+readonly workshop_inventory="/home/${workshop_user}/hosts.ini"
+readonly workshop_ansible_config="/home/${workshop_user}/.ansible.cfg"
+
+readonly workshop_git_repo="https://github.com/ansible/lightbulb.git"
+
+#
+# Functions
+#
+
+main() {
+  info "Bootstrapping Ansible on host ${HOSTNAME}."
+
+  ensure_ansible_installed
+  install_vagrant_private_key
+  configure_ansible
+  install_inventory
+  install_lightbulb
+
+}
+
+# Ansible installation
+ensure_ansible_installed() {
+  if ! is_ansible_installed; then
+    distro=$(get_linux_distribution)
+    "install_ansible_${distro}" || die "Distribution ${distro} is not supported"
+  fi
+
+  info "Ansible version"
+  ansible --version
```

Comment:
This fails, because ansible is not being installed during the flow.

Reply:
Right -- that's the line I hacked out to test the rest of this. I trust @dagwieers and @bertvv will pull that out when they get around to addressing the issues that have been raised.